packages/react-hooks/src/useThresholds.ts

```diff
@@ -18,6 +16,49 @@ export type ThresholdApi = {
   slashProposalThreshold: number;
 }
 
+const PROPOSE_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 1,
+  [KUSAMA_GENESIS]: 0.5,
+  [POLKADOT_GENESIS]: 0.6,
+  default: 0.5
+};
+
+const REJECT_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 0.5,
+  [KUSAMA_GENESIS]: 0.5,
+  [POLKADOT_GENESIS]: 0.5,
+  default: 0.5
+};
+
+const SLASH_THRESHOLDS: Record<string, number> = {
+  [KUSAMA_GENESIS]: 0.5,
+  [POLKADOT_GENESIS]: 0.75,
+  default: 0.5
+};
+
+const TREASURY_THRESHOLDS: Record<string, number> = {
+  [KULUPU_GENESIS]: 0.5,
+  [KUSAMA_GENESIS]: 0.6,
+  [POLKADOT_GENESIS]: 0.6,
+  default: 0.6
+};
+
+function getProposalThreshold (api: ApiPromise): number {
+  return PROPOSE_THRESHOLDS[api.genesisHash.toHex()] || PROPOSE_THRESHOLDS.default;
+}
+
+function getSlashProposalThreshold (api: ApiPromise): number {
+  return SLASH_THRESHOLDS[api.genesisHash.toHex()] || SLASH_THRESHOLDS.default;
+}
+
+function getTreasuryProposalThreshold (api: ApiPromise): number {
+  return TREASURY_THRESHOLDS[api.genesisHash.toHex()] || TREASURY_THRESHOLDS.default;
+}
+
+function getTreasuryRejectionThreshold (api: ApiPromise): number {
+  return REJECT_THRESHOLDS[api.genesisHash.toHex()] || REJECT_THRESHOLDS.default;
+}
```

Comment:
Since we do this in the hook, we probably don't even need the functions, can just add the one-liners in the hook.